library/Notifications/Widget/ItemList/EscalationConditionList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/* Icinga Notifications Web | (c) 2023 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Notifications\Widget\ItemList;
+
+use ipl\Html\BaseHtmlElement;
+
+class EscalationConditionList extends BaseHtmlElement
+{
+    protected $defaultAttributes = ['class' => 'options'];
+
+    protected $tag = 'ul';
+
+    /** @var EscalationConditionListItem[] Condition list items */
+    protected $conditions;
+
+    /**
+     * Create conditions list of the escalation
+     *
+     * @param EscalationConditionListItem[] $conditions
+     */
+    public function __construct(array $conditions)
+    {
+        $this->conditions = $conditions;
+    }
+
+    protected function assemble(): void
+    {
+        $removedPosition = null;
+        foreach ($this->conditions as $position => $condition) {
+            if ($condition->hasBeenRemoved()) {
+                $removedPosition = $position;
+
+                continue;
+            }
+
+            if ($removedPosition) {
```

Comment:
Just for a litte bit of safety: (I know positions start at 1)

```suggestion
            if ($removedPosition !== null) {
```